docs/rules/accessor-pairs.md

```diff
@@ -0,0 +1,150 @@
+# Disallow setters without getters in objects (accessor-pairs)
```

Comment:
The title is a bit misleading - maybe "Enforces getter/setter pairs" is more accurate?